sdk/go/x/auto/example_test.go

```diff
@@ -36,7 +35,7 @@ import (
 	"github.com/pulumi/pulumi/sdk/v2/go/x/auto/optup"
 )
 
-func Example() error {
+func Example() {
```

Comment:
TIL that Godoc examples don't return errors.. My IDE was complaining about them but seems linter doesn't care? I can kill these changes if you prefer.

Reply:
I think this makes sense, handling errors will really be up to the user anyway 👍